swiftUIHW/swiftUIHW/ContentView.swift

```diff
@@ -6,16 +6,136 @@
 //
 
 import SwiftUI
+import AVFoundation
 
+/// Экран плеера
 struct ContentView: View {
+    private enum Constants {
+        static let nextButtonText = "Next"
+        static let playButtonText = "Play"
+        static let stopButtonText = "Stop"
+        static let prevousButtonText = "Previous"
+        static let valueFormat = "%05.2f"
+        static let downloadButtonText = "Download"
+        static let downloadAlertPrimaryButtonText = "Вы хотите скачать трек?"
+        static let downloadText = "Скачать"
+        static let repostText = "Поделиться"
+        static let repostTitleText = "Вы хотите поделиться?"
+        static let imageFrameWidthValue = 300.0
+        static let imageFrameHeightValue = 500.0
+        static let buttonFrameWidth = 75.0
+        static let buttonFrameHeight = 50.0
+        static let buttonCornerRadius = 10.0
+        static let defaultButtonFrameWidth = 100.0
+        static let defaultButtonFrameHeight = 50.0
+        static let secondsInMinut = 60.0
+    }
+    
+    @ObservedObject private var viewModel = PlayerViewModel()
```

Comment:
```suggestion
    @ObservedObject private var viewModel = PlayerViewModel()
```
private значит ниже var body